local-install-siva-java.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
You are right: all, enry-java and siva-java will be replaced by ones from jitpack.io eventually, in spirit of https://github.com/src-d/berserker/pull/33

But right now it's not possible with neither siva-java (due to https://github.com/src-d/siva-java/issues/5) nor with enry-java (due to final artifact not including native dependencies https://github.com/src-d/backlog/issues/973).

So, would I suggest to keep building from source here for now not to block on this, until both above issues are resolved and jars are published. 

Then in a subsequent PR, source build of all dependencies together will be replaced to jitpack artifacts.
This work is tracked under https://github.com/src-d/berserker/issues/30